src/pep8ext_naming.py

```diff
@@ -412,3 +433,10 @@ def _extract_names(assignment_target):
         else:
             yield assignment_target.name
         return
+
+
+def is_mixed_case(name):
+    stripped_name = name.strip('_')
+    head = stripped_name[:1]
+    tail = stripped_name[1:]
+    return head.islower() and tail and not (tail.islower() or tail.isdigit())
```

Comment:
Actually this does not return boolean yet, Consider the case where name is 'a'...

Reply:
Ah, right, because of the `tail` test. You could wrap it all in `bool()`, but it's not super important.

Reply:
Fixed this #70.